sql-cli/examples/_dags/advanced.py

```diff
@@ -2,11 +2,11 @@
 from airflow import DAG
 from airflow.utils import timezone
 from astro import sql as aql
-from astro.table import Table
+from astro.sql.table import Table
 
 with DAG(
     dag_id="advanced",
-    start_date=timezone.parse("2022-10-05 06:47:15.790544"),
+    start_date=timezone.parse("2022-10-10 09:11:57.308687"),
```

Comment:
Why this change?

Reply:
This is the time of the last auto-generated example. I ran it because of the change you see below. Let's continue the discussion in https://github.com/astronomer/astro-sdk/pull/1032#discussion_r991511811.